src/gpg/fn/print_key.py

```diff
@@ -0,0 +1,117 @@
+# Print the details of the given key id.
+import os
+import sys
+import textwrap
+import time
+
+import gpg
+
+
+def print_key(identity):
+    """
+    print the details of key whose identitiy is passed
+    """
+    c = gpg.Context()
+    key = list(c.keylist(identity, mode=gpg.constants.keylist.mode.SIGS))
+
+    # exit with status 1 if more than one key matches are found for identity
+    if len(key) > 1:
+        exit(1)
+    else:
+        key = key[0]
+
+    if os.environ["DEBUG"] == "yes":
+        print(key)
+        print()
+
+    # uid
+    uid_list = ["uid: " + user_id.uid for user_id in key.uids]
+    all_uids = "\n".join(uid_list)
+
+    # fpr
+    fpr = " ".join(textwrap.wrap(key.fpr, 4))
```

Comment:
Keep formatting content together where it's used, for ease of editing. 
Move line 28,29,32 to 91.